sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/Filter.cs

```diff
@@ -235,6 +236,13 @@ internal static Type GetFieldType(string fieldName, out FieldNameType fieldNameT
             return GetPropertyTypeFromFieldName(fieldName);
         }
 
+        private static Expression CreateListAccessExpression(ParameterExpression documentExpression, string listName, MethodInfo tryGetValueMethodInfo, Type valueType, string keyValue)
+        {
+            // return Filter<int>.TryGetString(document.listName, keyValue)
```

Comment:
Why this is commented here?

Reply:
The reflection expression is hard to understand. So I added what it does in plain C# just like the following method (`CreateDictionaryAccessExpression`), which is even more tricky because it calls a function with an `out` parameter.